astro/src/content/docs/get-started/core-concepts/collected-metrics.mdx

```diff
@@ -0,0 +1,535 @@
+---
+title: Collected Metrics
+description: An overview of how, when, and why FusionAuth collects deployment usage data.
+section: get started
+subcategory: download and install
+---
+
+One of our goals is constant improvement. In order to get there, we collect some usage data from FusionAuth deployments. This data helps us understand how you're using FusionAuth so that we can give you better features, tightened security, and tailored experiences.
+
+Currently FusionAuth sends a payload to the metrics endpoint for all instances.
+
+Collected metrics of this payload include:
+
+-  a unique instance identifier
+-  the total number of users
+-  the total number of monthly active users
+-  the total number of monthly logins
+-  the number of nodes in the instance
+-  the product version
+-  the license Id (only for licensed instances with a plan)
+-  the version of the metrics data (currently always version 1)
+
+A self-hosted community user can disable network egress or block access to the metrics endpoint without affecting functionality.
+
+Customers on a paid plan must allow a metrics connection [per the license agreement](/license), unless they have an air-gapped license.
+
```

Comment:
```suggestion
### Detailed Metrics

<Aside type="version">
Available since 1.54.0
</Aside>
```

(you'll need to import the aside component as well)